libtmux/common.py

```diff
@@ -13,6 +13,7 @@
 import typing as t
 from distutils.version import LooseVersion
 from typing import Dict, Generic, KeysView, List, Optional, TypeVar, Union, overload
+from shutil import which as which
```

Comment:
Nitpick: Fine to `import shutil` and use `shutil.which()`

Reply:
Looks good!

Reply:
Was reverted by one of us during cross force push

I let you remove it